remote-api/src/test/java/org/alfresco/rest/api/discovery/DiscoveryApiWebscriptUnitTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * #%L
+ * Alfresco Repository
```

Comment:
Should probably rename this to `Alfresco Remote API`

Reply:
Done